pkgs/_cfe_macros/lib/macro_implementation.dart

```diff
@@ -121,9 +121,29 @@ class CfeRunningMacro implements injected.RunningMacro {
 /// functionality of `MacroExecutionResult`.
 class CfeMacroExecutionResult implements injected.MacroExecutionResult {
   final MacroTarget target;
-  final AugmentResponse augmentResponse;
-
-  CfeMacroExecutionResult(this.target, this.augmentResponse);
+  @override
+  final Map<Identifier, Iterable<DeclarationCode>> typeAugmentations;
+
+  CfeMacroExecutionResult(this.target, Iterable<DeclarationCode> declarations)
+      // TODO(davidmorgan): this assumes augmentations are for the macro
+      // application target. Instead, it should be explicit in
+      // `AugmentResponse`.
+      : typeAugmentations = {
+          // TODO(davidmorgan): empty augmentations response breaks the test,
+          // it's not clear why.
+          if (declarations.isNotEmpty)
+            (target as Declaration).identifier: declarations
+        };
+
+  static Future<CfeMacroExecutionResult> expandTemplates(
```

Comment:
This isn't really expanding templates any more right? I would rename it.

It seems like really this is translating between the DartModel code objects and package:macros Code objects?